CollectionsInterface/Exercise5/src/Car.java

```diff
@@ -0,0 +1,29 @@
+import java.util.Objects;
+
+public class Car {
+    private String model;
+    private Integer prize;
```

Comment:
Para manejar dinero, se tiende a utilizar la clase `BigDecimal`
```suggestion
    private BigDecimal prize;
```

Reply:
Ahah, ahí lo cambio. Eso ni lo pensé porque en la consigna decía Integer jaja pero tiene más sentido así